src/alr/alr-commands-pin.adb

```diff
@@ -214,14 +242,20 @@ package body Alr.Commands.Pin is
    function Long_Description (Cmd : Command)
                               return Alire.Utils.String_Vector is
      (Alire.Utils.Empty_Vector
-      .Append ("Pin releases to their current solution version."
+      .Append ("Pin releases to a particular version."
+               & " By default, the current solution version is used."
                & " A pinned release is not affected by automatic updates.")
       .New_Line
       .Append ("Without arguments, show existing pins.")
       .New_Line
       .Append ("Use --all to pin the whole current solution.")
       .New_Line
       .Append ("Specify a single crate to modify its pin.")
+      .New_Line
+      .Append ("Use the --url switch to "
```

Comment:
OK, this was something that made me uneasy too.

Reply:
Given that this will ripple through all tests and follow-up PRs... I was thinking to rename it to `--use` instead to avoid latter having two similar/aliased switches. What do you think?

Reply:
`alr pin --use=~/src/plop`?

Yeah I think that's ok.